### DISCUSSION TITLE
Display Trend indicator

### DISCUSSION THREAD

**Discussion by inomn** [vitest-dev/vitest#3827] [Q&A] [answered]
Hi! Is it possible to display a Trend indicator (diff) related to current PR?  ⬆️ \ ⬇️

Example: 
![image](https://github.com/vitest-dev/vitest/assets/6470994/9b019342-8a96-4430-a72e-ce332438bbbc)

Thanks.

**Comment by AriPerkkio** [comment, marked as answer]
Is this question related to [`vitest-coverage-report-action`](https://github.com/davelosert/vitest-coverage-report-action) and not Vitest itself? You might want to open a new issue there.

**Reply by inomn** [reply]
Thanks. You are right. Issue: https://github.com/davelosert/vitest-coverage-report-action/issues/262
